app/models/gsa18f/procurement.rb

```diff
@@ -76,7 +80,13 @@ def name
     end
 
     def self.approver_email
-      ENV['GSA18F_APPROVER_EMAIL'] || '18fapprover@gsa.gov'
+      raise 'No 18F approver was set' unless ENV['GSA18F_APPROVER_EMAIL']
+      ENV['GSA18F_APPROVER_EMAIL']
+    end
+
+    def self.purchaser_email
+      raise 'No 18F purchaser was set' unless ENV['GSA18F_PURCHASER_EMAIL']
```

Comment:
Thoughts on pulling out this email as an environment variable like we do for the approver? Also noticed that the default email for approver was a bogus email so what if we simply raise an error if the env var isn't set?